cmd/generate.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"io"
+
+	rm "github.com/codilime/floodgate/resourcemanager"
+	"github.com/spf13/cobra"
+)
+
+// generateOptions store render command options
+type generateOptions struct {
+	outDir string
+}
+
+// NewGenerateCmd create new generate command
+func NewGenerateCmd(out io.Writer) *cobra.Command {
```

Comment:
I am not sure if we want to have such a command. In my opinion config should be automatically generated (if at all) in default location if floodgate is run, there is no config provided and there is no config in default location. There should also be a note for the user that the config was generated in default location and the password field needs to be changed.

Another idea: we should figure out some default location for resources and libraries as well and provide the possibility to pass a parameter with location of resources directory and password to Spinnaker.

However if we decide to keep this command, I think displaying the message for user is still appropriate and there should be an option at least for providing password during generation and default password should be something like `<enter_admin_password_here>`